grayskull/strategy/config.yaml

```diff
@@ -450,3 +450,8 @@ pywin32:
   conda_forge: pywin32
```

Comment:
Shouldn't this be pywin32-on-windows by the way: https://github.com/conda-forge/pywin32-on-windows-feedstock

Reply:
indeed! can you fix that in this PR as well please?

Reply:
Done! :)